tests/gui/test_inference_gui.py

```diff
@@ -153,3 +153,27 @@ def test_inference_merging():
     assert len(labels[2].user_instances) == 1
     # Only predicted instances with graphable points should be merged
     assert len(labels[2].predicted_instances) == 2
+
+
+def test_inference_movenet_cli(movenet_video):
+
+    models = ["movenet-lightning", "movenet-thunder"]
+
+    for model in models:
+
+        inference_task = runners.InferenceTask(
+            trained_job_paths=[model],
+            inference_params={"tracking.tracker": None},
+        )
+
+        item_for_inference = runners.VideoItemForInference(
+            video=movenet_video, frames=[1, 2, 3]
+        )
+
+        cli_args, output_path = inference_task.make_predict_cli_call(item_for_inference)
```

Comment:
Neat! Good enough for me, although, if we wanted the test coverage, we could call `LearningDialog.run()` from `test_movenet_selection` and use that test to replace both? `test_movenet_inference` and/or `test_movenet_predictor` - which could in turn decrease test time, but couple tests
https://github.com/talmolab/sleap/blob/8287cea0d6a1a79a6e72d7aca46ed02d91b8787f/tests/nn/test_inference.py#L1391-L1408
https://github.com/talmolab/sleap/blob/8287cea0d6a1a79a6e72d7aca46ed02d91b8787f/tests/nn/test_inference.py#L1411-L1431